### DISCUSSION TITLE
For "App Layouts", how to set windows size to 100%

### DISCUSSION THREAD

**Discussion by HackettLai** [rxhanson/RectanglePro-Community#547] [Q&A]
![CleanShot 2024-08-19 at 17 21 31](https://github.com/user-attachments/assets/cffd8e0b-59b0-4d84-81ce-87a3f8d98876)

Is there any way I can set the value in percentages so that I don't have to measure it in pixels?

**Comment by Aaron-Rumpler** [comment]
As per the popup:
> Numeric fields can be:
> - fractions, whole, or decimal
> - ≤ 1 for % of screen
> - \> 1 for pixels • left blank to keep current window dimension

So for example, 100% would be 1, 50% would be 0.5, 25% would be 0.25, etc.
